src/Node/ChainInfo.ts

```diff
@@ -37,6 +37,12 @@ export default class ChainInfo {
    * @returns Chain id; based on the given input.
    */
   public static getChainId(chain: string): string {
+    if (chain === 'origin' || chain === 'dev') {
```

Comment:
We should document significance of 1515 and 1000.

Reply:
We can identify dev chains with identifier devChain and devAuxiliary.

Identifiers origin, dev, auxiliary are confusing 😕 
Same comment at other places also.